### ISSUE TITLE
Консоль в ксенобиологии

### ISSUE THREAD

**Issue by XXXpornXXX** [ITBlackwood/ATMTA-Station#106]
**Описание проблемы**:
Консоль у ксенобиолога через некоторое время работ с мартышками перестаёт их поднимать.

**Что должно было произойти**: 
Должно было произойти поднятие мартышек, но консоль перестала их поднимать через некоторое время.

**Что произошло вместо этого**:
Ничего.

**Почему это плохо/Какие последствия этого:**
Приходится ходить за мартышками самому, а обычно это неудобно, ведь можно просто пособирать трупы и оставить у перерабатывателя. Последствия такие, что неприручённые слаймы могут легко укусить и сломать кость.

**Детальное описание как повторить возникновение проблемы**:
1.Стать ксенобиологом.
2.Подойди к консоли и начать с ней работать
3. Через некоторое время консоль просто перестанет поднимать мартышек. 
4. Fin
**Что-нибудь еще что может помочь (какой был режим, была ли у вас роль, какие-то особенности и т.д.)**: 
Я был обычным ксенобиологом.

**Comment by ITBlackwood** [comment]
Лимит?

**Comment by Dydna** [comment]
Нет, даже если в ней нет мартышек, она всё равно не будет ничего брать.

**Comment by Dydna** [comment]
Так, сообщаю, это всё из-за трупов мартышек, если использовать её только с кубами и живыми мартышками, то всё работает больше 4х часов. Учти, что именно поднятие трупов вызывает этот баг, @ITBlackwood
